pages/common/csvlook.md

```diff
@@ -0,0 +1,16 @@
+# csvlook
+
+> Render a CSV file in the console as a fixed-width table.
+> Included in csvkit.
+
+- View a CSV file:
+
+`csvlook {{data.csv}}`
+
+- View a CSV file with _less_ for easy scrolling:
```

Comment:
I guess we can cut it down to just the trivial example. It doesn't hurt to have the page, however simple.